src/test/fuzz/crypto_diff_fuzz_chacha20.cpp

```diff
@@ -43,10 +43,10 @@ typedef unsigned char u8;
 /* ------------------------------------------------------------------------- */
 /* Data structures */
 
-typedef struct
+using ECRYPT_ctx = struct
```

Comment:
In commit "refactor: use "using" over "typedef"" (f5284d39a2b78bb07655c397386b9741495b6d41)

This one is a little weird because it was a C-style struct typedef. Should probably just be updated `struct ECRYPT_ctx {`

Reply:
Applied this suggestion.